web3.js/src/connection.js

```diff
@@ -1872,6 +1903,35 @@ export class Connection {
     return result.result;
   }
 
+  /**
+   * Fetch a list of all the confirmed signatures for transactions involving an address
+   * within a specified slot range. Max range allowed is 10,000 slots.
+   *
+   * @param address queried address
+   * @param before start searching backwards from this transaction signature.
+   *               If not provided the search starts from the highest max confirmed block.
+   * @param limit maximum transaction signatures to return (between 1 and 1,000, default: 1,000).
+   */
+  async getConfirmedSignaturesForAddress2(
+    address: PublicKey,
+    before: ?TransactionSignature,
+    limit: ?number,
```

Comment:
I think an options object works better here. It's awkward when you need to do stuff like `connection. getConfirmedSignaturesForAddress2(pubkey, undefined, 100)`

How about this:
```js
export type ConfirmedSignaturesOptions = {
  before: ? TransactionSignature,
  limit: ?number,
};
```

So that the call becomes: `connection.getConfirmedSignaturesForAddress2(pubkey, {limit: 100})`

Reply:
Hmm, yeah.  Why didn't I do that!  The RPC method does it that way even.  Fixed now